post-scan-actions/aws-python-promote-or-quarantine/handler.py

```diff
@@ -48,11 +79,22 @@ def lambda_handler(event, context):
 
         findings = message['scanning_result'].get('Findings')
 
-        if not findings:
-            copy_object(source_bucket=src_bucket, dest_bucket=promote_bucket, source_key=object_key, dest_key=object_key)
-            delete_objects(bucket=src_bucket, prefix='', objects=[object_key])
-            print('Promoted file successfully')
-        else:
-            copy_object(source_bucket=src_bucket, dest_bucket=quarantine_bucket, source_key=object_key, dest_key=object_key)
+        mode = quarantine_mode if findings else promote_mode
+        dst_bucket = quarantine_bucket if findings else promote_bucket
+
+        if not dst_bucket:
+            print(f'Skip: No bucket specified for {"quarantine" if findings else "promotion"}')
+            continue
+
+        copy_object(
+            source_bucket=src_bucket,
+            dest_bucket=dst_bucket,
+            source_key=object_key,
+            dest_key=object_key,
+            acl=acl,
+        )
+
+        if mode == 'move':
             delete_objects(bucket=src_bucket, prefix='', objects=[object_key])
-            print('Quarantined file successfully')
+
+        print(f'File {"quarantined" if findings else "promoted"} successfully (mode: {mode})')
```

Comment:
Follow the previous refactoring comments.

```suggestion
        print(f'File {operation} successfully (mode: {mode})')
```